docker-compose.yml

```
@@ -0,0 +1,20 @@
+version: "3"
+
+services:
+  tests: &tests
+    image: keboola/dev-notification-api-php-client
+    build:
+      context: .
+      dockerfile: Dockerfile
+    volumes:
+      - ./build/logs/:/code/build/logs/
+    environment:
+      TEST_NOTIFICATION_API_URL:
+      TEST_STORAGE_API_TOKEN:
+      TEST_MANAGE_API_APPLICATION_TOKEN:
```

Nejsem si jisty, ale tohle preda ty ENV promenny vzdyky prazdny ne? Nechtel si to udelat jako seznam?

```suggestion
    environment:
      - TEST_NOTIFICATION_API_URL
      - TEST_STORAGE_API_TOKEN
      - TEST_MANAGE_API_APPLICATION_TOKEN
```

---

Je to jedno https://docs.docker.com/compose/compose-file/compose-file-v3/#environment